src/overlays/actors/ovl_En_Suttari/z_en_suttari.c

```diff
@@ -71,114 +126,1434 @@ static DamageTable D_80BAE7E0 = {
+    u16 sp26 = gSaveContext.time - 0x3FFC;
+    u16 pad1;
+    u8 sp23 = ENSUTTARI_GET_PATH(&this->actor);
+    u16 pad2;
+    UNK_TYPE sp1C = D_80BAE8F8[unkStruct->unk0];
+    u16 phi_a0;
+
+    if (sp1C >= 0) {
+        this->unk404 = func_8013BB34(globalCtx, sp23, sp1C);
+    }
+    if (this->unk404 == NULL) {
+        return 0;
+    }
+    if ((this->unk428 != 0 && this->unk428 < 0xC) && (this->unk42A >= 0)) {
+        phi_a0 = sp26;
+    } else {
+        phi_a0 = unkStruct->unk4;
+    }
+    if (unkStruct->unk8 < phi_a0) {
+        this->unk418 = (phi_a0 - unkStruct->unk8) + 0xFFFF;
+    } else {
+        this->unk418 = unkStruct->unk8 - phi_a0;
+    }
+    this->unk424 = sp26 - phi_a0;
+    phi_a0 = this->unk404->count - 2;
+    this->unk42C = 0;
+    this->unk41C = this->unk418 / phi_a0;
+    this->unk420 = (this->unk424 / this->unk41C) + 2;
+    this->unk430 = 0;
+    return 1;
+}
+
+s32 func_80BABDD8(EnSuttari* this, GlobalContext* globalCtx, struct_80133038_arg2* unkStruct) {
+    s32 pad;
+    EnDoor* sp48;
+    u8 sp47;
+    u16 sp44;
+    Vec3f sp38;
+    Vec3f sp2C;
+    Vec3s* sp28;
+    UNK_TYPE sp24;
+
+    sp47 = ENSUTTARI_GET_PATH(&this->actor);
+    sp44 = gSaveContext.time - 0x3FFC;
+    if (this->unk428 == 10 || this->unk428 == 11 || this->unk428 == 2) {
+        return 0;
+    }
+    sp48 = (EnDoor*)func_8013BB7C(&this->actor, globalCtx, ACTORCAT_DOOR, ACTOR_EN_DOOR);
+    sp24 = D_80BAE8F8[unkStruct->unk0];
+    if ((sp48 != NULL) && (sp24 >= 0)) {
+        this->unk404 = func_8013BB34(globalCtx, sp47, sp24);
+    }
+    if ((sp48 == NULL) || (this->unk404 == NULL)) {
+        return 0;
+    }
+    sp28 = (Vec3s*)Lib_SegmentedToVirtual(this->unk404->points);
+    Math_Vec3s_ToVec3f(&sp38, &sp28[0]);
+    Math_Vec3s_ToVec3f(&sp2C, &sp28[1]);
+    this->unk434 = sp44 - unkStruct->unk4;
+    this->unk436 = unkStruct->unk8 - unkStruct->unk4;
+    if (unkStruct->unk0 != 10 && unkStruct->unk0 != 11) {
+        sp48->unk1A7 = 0x4B;
+    }
+    Math_Vec3f_Copy(&this->unk438, &sp38);
+    Math_Vec3f_Copy(&this->unk444, &sp2C);
+    this->actor.world.rot.y = Math_Vec3f_Yaw(&sp38, &sp2C);
+    this->actor.world.pos = sp38;
+    return 1;
+}
+
+s32 func_80BABF64(EnSuttari* this, GlobalContext* globalCtx, struct_80133038_arg2* unkStruct) {
+    s32 ret;
+
+    switch (unkStruct->unk0) {
+        case 15:
+        case 14:
+        case 13:
+        case 12:
+            ret = func_80BABC48(this, globalCtx, unkStruct);
+            break;
+        case 11:
+        case 10:
+        case 9:
+        case 8:
+        case 7:
+        case 6:
+            ret = func_80BABDD8(this, globalCtx, unkStruct);
+            break;
+        case 5:
+        case 4:
+        case 3:
+        case 2:
+        case 1:
+            ret = 1;
+            break;
+        default:
+            ret = 0;
+            break;
+    }
+    return ret;
+}
+
+s32 func_80BABFD4(EnSuttari* this, GlobalContext* globalCtx) {
+    UNK_TYPE1 sp7C[0x424];
```

Comment:
I *think* from looking into the functions that it's used by that this is actually words. What sort of words I'm not sure...

Reply:
I'm also very curious as to what this is.